### DISCUSSION TITLE
unable to run these command "npm i && npm link" and "email-auth-node --version" in POWERSHELL

### DISCUSSION THREAD

**Discussion by BilalN10** [saran-surya/email_auth_node#50] [Q&A] [answered]
Hi. 
I'm not able to run these command "npm i && npm link" and "email-auth-node --version". Please help me I'm really in trouble. I just only a way to send OTP on email in Flutter .
I found this error when try to run this commmand 
PS C:\Users\Bilal\Desktop\eshop> npm i && npm link
At line:1 char:7
+ npm i && npm link
+       ~~
The token '&&' is not a valid statement separator in this version.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException

**Comment by saran-surya** [comment, marked as answer]
Yes, it is because you are running it using powershell, Kindly use CMD. It will be fine.
If you want to stick to powershell, then seperate the commands and execute them.
```powershell
npm i
npm link
```

## Also requesting you to change the server from google to outlook, as google has removed the access for less secure apps, Please pull the latest changes from the repo
